myst_nbs/gaussian_processes/GP-MaunaLoa.myst.md

```diff
@@ -500,30 +508,30 @@ This plot makes it clear that there is a broadening over time.  So it would seem
 
 ## What day will the CO2 level break 400 ppm?
 
-How well do our forecasts look?  Clearly the observed data trends up and the seasonal effect is very pronounced.  Does our GP model capture this well enough to make reasonable extrapolations?  Our "training" set went up until the end of 2003, so we are going to predict from January 2014 out to the end of 2017.  
+How well do our forecasts look?  Clearly the observed data trends up and the seasonal effect is very pronounced.  Does our GP model capture this well enough to make reasonable extrapolations?  Our "training" set went up until the end of 2003, so we are going to predict from January 2004 out to the end of current year.  
```

Comment:
```suggestion
How well do our forecasts look?  Clearly the observed data trends up and the seasonal effect is very pronounced.  Does our GP model capture this well enough to make reasonable extrapolations?  Our "training" set went up until the end of 2003, so we are going to predict from January 2004 out to the end of 2022.  
```

The notebook should read correctly after 2022 finishes. Because it might not be updated before 2023 but also because we generate snapshots of the examples every few months so that books and the like can cite stable website and the snapshots could perfectly be read after 2022 and won't be updated, not even a comma